### ISSUE TITLE
gelf: support HTTP authentication

### ISSUE THREAD

**Issue by arvidvillen** [quarkusio/quarkus#9355]
**Description**
When configuring _Amazon Elasticsearch Service_ with public access and you don't want to configure IP based security it is recommended to use HTTP authentication and it would be nice if `quarkus-logging-gelf` supported that.

**Implementation ideas**
Even if its not very clear in the documentation it seems that the extension supports HTTPS by setting `quarkus.log.handler.gelf.host` to something like https://example.com/, however adding the credentials like https://username:password@example.com/ does not work.

**Additional note**
See https://github.com/mp911de/logstash-gelf/issues/194

**Comment by loicmathieu** [comment]
@arvidvillen the `logstash-gelf` library didn't support HTTP authentication yet so I put this enhancement on ice untill it supports basic auth, then we can have a look at supporting it (and documenting the HTTP support).

**Comment by gsmet** [comment]
Has a feature request been open for `logstash-gelf`?

**Comment by loicmathieu** [comment]
Yes, it is already linked in the description and apparently @arvidvillen is working on it so as soon as it's released we can un-freeze this one and implement it.

**Comment by loicmathieu** [comment]
The enhancement has been done on the upstream library, it will be available on 1.15.0 wich is not yet publshed.

**Comment by loicmathieu** [comment]
Hi @arvidvillen the new version of `logstash-gelf` has been integrated in Quarkus and support HTTP authentication.

Will you still willing to implement this for Quarkus ?
